src/FakeItEasy/Core/FakeManager.PropertyBehaviorRule.cs

```diff
@@ -39,25 +39,18 @@ public static bool IsPropertyGetter(MethodInfo method)
                 return method.IsSpecialName && method.Name.StartsWith("get_", StringComparison.Ordinal);
             }
 
+            public bool IsMatchForSetter(IFakeObjectCall fakeObjectCall) => IsPropertySetter(fakeObjectCall);

```

Comment:
I don't think it's worth renaming.
The only alternative     I can think of that I like is `IsPropertySetter`. :wink: